ct-app/core/model/peer.py

```diff
@@ -134,9 +152,11 @@ def apr_percentage(self):
         seconds_in_year = 60 * 60 * 24 * 365
         period = self.economic_model.budget.period
 
-        return (
-            (self.expected_reward * (seconds_in_year / period)) / self.split_stake
-        ) * 100
+        apr = (
+            (self.expected_reward / self.split_stake) * (seconds_in_year / period) * 100
+        )
+
+        return apr if self.max_apr is None else min(self.max_apr, apr)
```

Comment:
It'd actually be more readable to turn the logic upside down, i.e assume that the `self.max_apr` value can never be `None`:

```python
self.max_apr = float("inf")       # or sys.float_info.max

...

return min(self.max_apr, apr)
```